.kokoro/presubmit/presubmit.cfg

```diff
@@ -9,3 +9,11 @@ env_vars: {
     key: "RUN_SNIPPETS_TESTS"
     value: "false"
 }
+env_vars: {
+    key: "RUN_TYPING_TESTS"
```

Comment:
This PR proposes we effectively double our RUN_* boolean knobs from 2 to 4, which to me feels we have an opportunity to improve our approach further.  I like that NOX_SESSION allows us to be explicit with a list of sessions rather than a single session.

For the general case, should we consolidate the control knobs to something more expressive like some kind of session group filtering syntax?  I think we're mostly concerned about the cases where NOX_SESSION isn't set.

If we maintain a map of filtername -> list of sessions, we could potential skip sessions if a single filter variable is set.

Reply:
Not gonna lie:

I have very little idea how to this style of config files (both here and on google3) are supposed to be formatted and how they interact.

I spent more time than I should in just trying to get to this point.

I agree whole heartedly that I want to have better knobs.

The RUN_* knobs are part of a bigger/different picture:
* As we discussed in a meeting, we create the virtual envs using a nox.session decorator and then we come back and check to see if we even need to run the session at all by checking these knobs, which wastes time making the virtual_envs needlessly

But in the interest of getting something that is functional, keeping this PR small and clean and can prove that we can add a new parallel kokoro job, I chose to aim for an approach that we already use and that works, knowing that per our CI/CD redesign, I was gonna come back and figure out whether we can improve the knob situation holistically.

I am not sure what you mean by these statements: I am not picturing your end state... can you elaborate?

> For the general case, should we consolidate the control knobs to something more expressive like some kind of session group filtering syntax? I think we're mostly concerned about the cases where NOX_SESSION isn't set.

> If we maintain a map of filtername -> list of sessions, we could potential skip sessions if a single filter variable is set.